static-site/src/sections/sars-cov-2-page.jsx

```diff
@@ -22,9 +21,6 @@ import GenericPage from "../layouts/generic-page";
 import { ErrorBanner } from "../components/splash/errorMessages";
 import { withRouter } from 'next/router'
 
-const nextstrainLogoPNG = "/favicon.png";
```

Comment:
Suggestion: remove the favicon.png file along with this change. I don't see any other references.

Reply:
Thanks for picking this up. I've decided to move the favicons from the gatsby public folder (`./static-site/static`) to the next.js public folder (`./static-site/public`) which allows us to drop the `serve-favicon` dependency I originally added to this PR and use the implicit approach we were using with Gatsby.

Reply:
Looks like it's still unused though? At least on `sars-cov-2-page.jsx`, the reference changed from `favicon.png` to `nextstrain-logo-small.png`.

Reply:
It's unused in our code, but browsers will (unless instructed otherwise) access it to use as a favicon. This [post](https://webmasters.stackexchange.com/questions/123109/can-a-favicon-at-the-root-path-be-in-a-different-image-format-than-ico) is a good summary, and you can see it in the network requests too:

<img width="514" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/5de03117-3a8c-4e69-a754-066efbd09ad0">

Reply:
Are you confusing `favicon.png` with `favicon.ico`? I see that the ICO is used by default unless instructed otherwise to use PNG. I only get a network request to PNG after adding the line in the post you linked using 203440f334e2c1c3336b1942528da67a8d777e01.

Reply:
> unless instructed otherwise to use PNG

I thought some browsers accessed PNG by default but looking into it I think that's not correct. I'll remove the PNG version now.

Reply:
[Removed](https://github.com/nextstrain/nextstrain.org/compare/e54146066179d2a6a52a8a554c5310ca99b89802..9668e99897fadc9d8f2a761bc0229755f1deba40)

Reply:
We both missed that Auspice requests the PNG version. Unlucky! The file is restored in  https://github.com/nextstrain/nextstrain.org/pull/825

Reply:
Oops, thanks for catching!